packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -5,11 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [9.7.0] - 2021-10-05
```

Comment:
Should we not add something about fixing this tool for Windows cmd?

Reply:
Forgot 😮‍💨